content/millennium/r4/general-clinical/allergy-intolerance.md

```diff
@@ -0,0 +1,111 @@
+
+The following fields are returned if valued:
+
+* [Allergy id](http://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Allergy code](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.code){:target="_blank"}
+* [clinicalStatus](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.clinicalStatus){:target="_blank"}
+* [verificationStatus](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.verificationStatus){:target="_blank"}
+* [Patient with allergy/intolerance](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.patient){:target="_blank"}
+* [Date of allergy onset](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.onset){:target="_blank"}
+* [Date/Time recorded](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.recordedDate){:target="_blank"}
+* [Who recorded](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.recorder){:target="_blank"}
+* [Who reported](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.asserter){:target="_blank"}
+* [Criticality/potential harm](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.criticality){:target="_blank"}
+* [Category (medication, food, environment)](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.category){:target="_blank"}
+* [Adverse reaction (clinical symptoms/manifestation)](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.reaction){:target="_blank"}
+* [Comment](http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.reaction.note){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:allergy_intolerance, :r4) %>
+
+## Search
+
+Search for AllergyIntolerances that meet supplied query parameters:
+
+    GET /AllergyIntolerance?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name               | Required?                                                            | Type          | Description
+--------------------|----------------------------------------------------------------------|---------------|-----------------------------------------------------------------------
+ `_id`              | This or `patient`, if populated all other parameters are not allowed | [`token`]     | The logical resource id associated with the resource. Example: `12345`
+ `patient`          | This or `_id`                                                        | [`reference`] | Who the sensitivity is for. Example: `12345`
+ `clinical-status`  | No                                                                   | [`token`]     | The clinical status of the allergy or intolerance. Example: active | inactive | resolved
```

Comment:
These need to be escaped, otherwise it will create new columns for each status:

```suggestion
 `clinical-status`  | No                                                                   | [`token`]     | The clinical status of the allergy or intolerance. Example: active \| inactive \| resolved
```